src/commands/new.ts

```diff
@@ -389,30 +391,25 @@ export default {
       overwrite,
     })
     stopSpinner(" 3D-printing a new React Native app", "🖨")
+    // copy the .gitignore if it wasn't copied over
+    // Release Ignite installs have the boilerplate's .gitignore in .gitignore.template
+    // (see https://github.com/npm/npm/issues/3763); development Ignite still
+    // has it in .gitignore. Copy it from one or the other.
+    const boilerplateIgnorePath = exists(boilerplate(".gitignore.template"))
+      ? boilerplate(".gitignore.template")
+      : boilerplate(".gitignore")
+    const targetIgnorePath = log(path(targetPath, ".gitignore"))
+    copy(log(boilerplateIgnorePath), targetIgnorePath, { overwrite: true })
+
+    if (exists(targetIgnorePath) === false) {
+      warning(`  Unable to copy ${boilerplateIgnorePath} to ${targetIgnorePath}`)
+    }
 
     // note the original directory
     const cwd = log(process.cwd())
 
     // jump into the project to do additional tasks
     process.chdir(targetPath)
-
-    // copy the .gitignore if it wasn't copied over
-    // Release Ignite installs have the boilerplate's .gitignore in .gitignore.template
-    // (see https://github.com/npm/npm/issues/3763); development Ignite still
-    // has it in .gitignore. Copy it from one or the other.
-    const targetIgnorePath = log(path(boilerplatePath, ".gitignore"))
-    if (!exists(targetIgnorePath)) {
```

Comment:
I suspect this is the offending logic here. This file isn't in the global NPM package artifacts. So it will never get copied over in production.

However, in development this actually exists